pkg/to/value.go

```diff
@@ -28,6 +28,19 @@ func String(s *string) string {
 	return ""
 }
 
+// Map array of string pointers to an array of strings. If a value in the the array is nil then append empty string.
+func StringArray(s []*string) []string {
+	var values []string
+	for _, ptr := range s {
+		if ptr != nil {
+			values = append(values, *ptr)
+		} else {
```

Comment:
This is an edge case, but I don't think we'd want to append empty strings if a value is nil. We should only be adding to the array if we get a managed identity string